packages/dnslink-api/src/index.js

```diff
@@ -16,8 +16,8 @@ const dnslinkSkylinkRegExp = new RegExp(`^dnslink=/${dnslinkNamespace}/([a-zA-Z0
 const hint = `valid example: dnslink=/${dnslinkNamespace}/3ACpC9Umme41zlWUgMQh1fw0sNwgWwyfDDhRQ9Sppz9hjQ`;
 
 server.get("/dnslink/:name", async (req, res) => {
-  const success = (skylink) => res.set("Skynet-Skylink", skylink).send(skylink);
-  const failure = (message) => res.status(400).set("Dnslink-Error", message).send(message);
+  const success = (skylink) => res.send(skylink);
+  const failure = (message) => res.status(400).send(message);
```

Comment:
The only reason I had them in header is because I previously used `http.request` from "socket.http" library and it would return buffered body so it wasn't easy to read from it and I had to inject that data to headers in dnslink-api. We've changed the http library now and we have full body response as expected so we don't need the same data in headers any more.